packages/shared/src/components/cards/MarketingCTA/common.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Button, ButtonSize, ButtonVariant } from '../../buttons/Button';
+import { MiniCloseIcon } from '../../icons';
+import { CardTitle } from '../Card';
+import classed from '../../../lib/classed';
+import { anchorDefaultRel } from '../../../lib/strings';
+
+export interface MarketingCTA {
+  variant: 'card' | 'popover';
+  tagText?: string;
+  tagColor?: string;
+  title: string;
+  description?: string;
+  image?: string;
+  ctaUrl: string;
+  ctaText: string;
+}
+
+type HeaderProps = Pick<MarketingCTA, 'tagText' | 'tagColor'>;
+const tagColorMap: Record<string, string> = {
+  avocado: 'bg-action-upvote-float text-action-upvote-default',
+};
+export const Header = ({ tagText, tagColor }: HeaderProps): ReactElement => (
+  <div className="flex w-full flex-row items-center">
+    <span
+      className={classNames(
+        'rounded-8  p-1 px-2 font-bold  typo-footnote',
```

Comment:
```suggestion
        'rounded-8 p-1 px-2 font-bold typo-footnote',
```

Reply:
Hmm surprised my linter doesn't catch this 😅